src/input.rs

```diff
@@ -29,14 +29,20 @@ pub fn is_key_down(key_code: KeyCode) -> bool {
     context.keys_down.contains(&key_code)
 }
 
-/// Detect if the key is being pressed.
-/// Each call "is_key_down" call will consume a character from input queue.
-pub fn get_key_pressed() -> Option<char> {
+/// Return the last pressed char.
```

Comment:
I think the part of consuming characters in the docs was quite important, I would like to keep it. There was a wrong name used in the docs though, but still :) 

Also I think the `get_char_pressed` was a good enough name

Reply:
ok like this
```rust
/// Return the last pressed char.
/// Each call "is_key_down" call will consume a character from input queue.
pub fn get_char_pressed() -> Option<char> {
.....
```

Reply:
Maybe:

```
Each "get_char_pressed" call will consume a character from the input queue.
```